MangoAPI.Client/src/app/components/chats/chats.component.ts

```diff
@@ -430,6 +461,12 @@ export class ChatsComponent implements OnInit, OnDestroy {
       });
   }
 
+  private fetchMessageImage(messageId: string, messageAttachmentUrl: string) {
```

Comment:
It is not necessary to download image explicitly, if attachment exists -- message contains link to it

See screenshot below for specs

![image](https://user-images.githubusercontent.com/31904595/216808442-c55fead4-9305-47d8-a811-6adb7c327987.png)

Reply:
I think better to use message Ids, because the source or route of blobs might change and all this links might be crushed.
for example if you wanna migrate from azure blob storage to s3